### DISCUSSION TITLE
Default surrounding line context shown in side-by-side reviews is overly stingy

### DISCUSSION THREAD

**Discussion by pozega** [community/community#56502] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When doing a code review in side by side mode, there are only 3 lines above and below the change shown as context.  This is insufficient to my needs to fully understand changes.

I would like to have this fixed to be larger, or customizable.  If customizable the setting can go here 
<img width="230" alt="image" src="https://github.com/community/community/assets/125319324/8fe5d7f5-35e5-409a-abf5-722bf7e72c2a">
Something like 'Show [x] lines above and below changes' where x is dropdown box for 3, 5, 10, 20.

**Comment by davidfstr** [comment]
I would also like the **number of lines of context** to be customizable, and in particular to be increased.
